### PR TITLE
Connection status bug

### PR DESCRIPTION
**Pull request by ggenya132**
In regards to, #38 

This impl of the session store fixes the issue. It is unideal and warrants a discussion of how cookies and session state will be used post MVP.

### CODE REVIEW

File: server/api/session-auth.js [hunk 1]
```diff
@@ -27,16 +27,7 @@ router.post(
...
-    (req, res, next) => {
-        const { channelId } = req;
-        updateConnectionStatusByChannelId(channelId, 'active')
-            .then((status) => {
-                console.log({ status });
-                next();
-            })
-            .catch(console.error);
-    }
+    sessionAuthController.connectToTwitchChat
```
Comment (SaxyPandaBear) [inline server/api/session-auth.js:30]: I was gonna mention this at some point, but this method name is not accurate. We aren't connecting to twitch chat. we're listening for pubsub events from the twitch api

File: server/lib/dynamoSessionStore.js [hunk 2]
```diff
@@ -0,0 +1,335 @@
+/*!
+ * Connect - DynamoDB
+ * Copyright(c) 2020 introvert.com LLC <support@introvert.com>
```
Comment (SaxyPandaBear) [inline server/lib/dynamoSessionStore.js:3]: We don't need their license :P this is our stuff now.

File: server/lib/dynamoSessionStore.js [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+/**
+ * Module dependencies.
+ */
+var AWS = require('aws-sdk');
+
+/**
+ * One day in milliseconds.
+ */
+
+var oneDayInMilliseconds = 86400000;
```
Comment (SaxyPandaBear) [inline server/lib/dynamoSessionStore.js:15]: this is a constant, so lets declare it so

File: server/lib/dynamoSessionStore.js (hunk 2 above)
Comment (ggenya132) [reply server/lib/dynamoSessionStore.js:3]: YES!

**Review by SaxyPandaBear — APPROVED** [review]

**Pull request merged**